L1Trigger/L1TMuonEndCap/src/PrimitiveConversion.cc

```diff
@@ -730,15 +786,35 @@ void PrimitiveConversion::convert_gem(
     double glob_phi   = emtf::rad_to_deg(gp.phi().value());
     double glob_theta = emtf::rad_to_deg(gp.theta());
     double glob_eta   = gp.eta();
+    double glob_rho   = gp.perp();
+    double glob_z     = gp.z();
+
 
     // Use the CSC precision (unconfirmed!)
-    int fph = emtf::calc_phi_loc_int(glob_phi, sector_, 13);
-    int th  = emtf::calc_theta_int(glob_theta, conv_hit.Endcap(), 7);
+    int fph = emtf::calc_phi_loc_int(glob_phi, conv_hit.PC_sector());
+    int th  = emtf::calc_theta_int(glob_theta, conv_hit.Endcap());
+
+    if (is_me0) {
+      // The ME0 chamber 1 starts at -10 deg. The CSC chamber 1 starts at -5 deg.
+      // This 5 deg difference unfortunately causes the local phi coord to go
+      // out of bound. This is because the local phi 0 is set to the CSC chamber
+      // edge minus 22 deg to accommodate for the neighbor chamber. However, it
+      // is possible for the ME0 neighbor chamber to go to as far as the CSC
+      // chamber edge minus 25 deg.
```

Comment:
So in this case you're just leaving the ME0 hit at phi = 0 in the chamber?  Or you discard it?

Reply:
It's being treated as 0 deg (in the local phi coord), even though it should be -3 deg. Not sure if there's a better way to do it...

Reply:
This is probably the best way to do it for now - either that or discard them.  (Both create problems down the line if you're not careful.)  But we can't change the EMTF integer phi scale for each sector, since that's what's actually in the FW.

Reply:
Right. But just to add, even if we discard them, we will only discard ME0 hits in the neighbor sector that are -25 to -22 deg away from the native sector. Discarding them should not affect the efficiency at the sector boundaries.